lib/pubsub/subscription.js

```diff
@@ -401,7 +401,7 @@ Subscription.prototype.pull = function(options, callback) {
  */
 Subscription.prototype.setAckDeadline = function(options, callback) {
   var body = {
-    ackId: options.ackId,
+    ackIds: util.arrayize(options.ackIds),
```

Comment:
Yeah, but pre-1.0, no need to worry about legacy support / documenting two cases. Thanks for reviewing, though :)